virttest/utils_misc.py

```diff
@@ -536,10 +536,13 @@ def get_path(base_path, user_path):
     :param base_path: The base path of relative user specified paths.
     :param user_path: The user specified path.
     """
-    if os.path.isabs(user_path) or aurl.is_url(user_path):
+    if aurl.is_url(user_path):
         return user_path
-    else:
-        return os.path.join(base_path, user_path)
+    if not os.path.isabs(user_path):
+        user_path = os.path.join(base_path, user_path)
+    if os.path.islink(user_path):
+        user_path = os.path.relpath(user_path)
```

Comment:
It should be os.path.realpath()?
>>> os.path.relpath("/usr/share/avocado/data/avocado-vt/images")
'../usr/share/avocado/data/avocado-vt/images'
>>> os.path.realpath("/usr/share/avocado/data/avocado-vt/images")
'/home/kvm_autotest_root/images'